Index.html

```diff
@@ -20,21 +20,140 @@
             <span class="bar"></span>
         </nav>
     </header>
-    <section class="section1">
-        <h1 id="greetings">I’m Alexandra. Glad to see you!</h1>
-        <p id="details">I’m a software developer! I can help you build a product , feature or website Look through
+    <section class="section">
+        <h1 >I’m Alexandra. Glad to see you!</h1>
+        <p>I’m a software developer! I can help you build a product , feature or website Look through
             some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate
             to contact me.</p>
     </section>
-    <footer>
-        <span> LET’S CONNECT</span>
-        <ul>
+    <section class="section">
+       
+        <img class="imgalin" src="./images/Portfolio.svg">
+        <h2>Tonic</h2>
+        <span class="sublinks">
+            <span>CANOPY</span>
+            <ul>
+                <li><a href="#">Bank End Dev</a></li> 
+                <li><a href="#">2015</a></li> 
+                          
+             </ul>
+        </span>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <span class="sublinksround">
+            
+            <ul>
+                <li><a href="#">HTML</a></li> 
+                <li><a href="#">CSS</a></li> 
+                <li><a href="#">JavaScript</a></li>          
+             </ul>
+        </span>
+      <a href=""class="seeproject" >See Project</a>
+    
+    </section>
+    
+    <section class="section">
+        <img class="imgalin" src="./images/Snapshoot2.svg">
+        <h2>Multi-Post Stories</h2>
+        <span class="sublinks">
+            <span>CANOPY</span>
+            <ul>
+                <li><a href="#">Bank End Dev</a></li> 
+                <li><a href="#">2015</a></li> 
+                          
+             </ul>
+        </span>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <span class="sublinksround">
+            
+            <ul>
+                <li><a href="#">HTML</a></li> 
+                <li><a href="#">CSS</a></li> 
+                <li><a href="#">JavaScript</a></li>          
+             </ul>
+        </span>
+      <a href="#" class="seeproject">See Project</a>
```

Comment:
- [ ] As requested by previous reviewers, kindly add a hover and pressed state to all your buttons as it is on the figma template. You can use the `:hover` and `:active` pseudoclasses for this.
![image](https://user-images.githubusercontent.com/94551484/227545089-c6ed521a-18ff-4bee-9e00-8f06897df79c.png)